.github/workflows/build.yml

```diff
@@ -52,3 +52,20 @@ jobs:
         kubectl apply -n argo -f manifests/mpi-operator.yaml
 
         scripts/integration_tests.sh
+  go-unittest:
```

Comment:
Thanks @Yancey1989 for setting this up. Is it possible to also include a golang format check in the CI?

Reply:
It's a good idea, I will do that later.